task/06-conditions-n-loops-tasks.js

```diff
@@ -81,8 +94,10 @@ function getSumBetweenNumbers(n1, n2) {
  *   10,1,1   =>  false
  *   10,10,10 =>  true
  */
-function isTriangle(a,b,c) {
-    throw new Error('Not implemented');
+function isTriangle(a, b, c) {
+    if (a + b > c && b + c > a && c + a > b) {
+        return true;
+    } else return false;
```

Comment:
We can  return (a + b > c) && (a + c > b) && (c + b > a) with the sam result
Please, improve logic for consistency
https://javascript.info/logical-operators